src/pageServices/contentScripts/selectable.js

```diff
@@ -216,16 +216,21 @@ export const selectable = () => {
       const selected = new Set();
       const deselected = new Set();
       const a = rb();
-      if (!a) {
-        return;
-      }
+
       delete self.ipos;
       document.body.classList.remove("s-noselect");
       document.body.removeEventListener("mousemove", self.rectDraw);
       document.body.removeEventListener("mouseup", self.select);
       document.body.removeEventListener("mouseleave", self.select);
 
       const s = self.options.selectedClass;
+
+      const removeRect = function () {
+        if (a && a.parentNode) a.parentNode.removeChild(a);
```

Comment:
Probably it’s better to rename variables like `a`, `s`, `e` to more descriptive names? Because now they look a bit confusing.

Reply:
agree with you! but this file is 90% a code that was pasted from a library and was changed just in a few places. That's why I don't want to refactor it